subsys/nrf_security/Kconfig

```diff
@@ -33,6 +33,7 @@ config NRF_SECURITY
 	bool
 	prompt "nRF Security" if !PSA_PROMPTLESS
 	depends on SOC_FAMILY_NORDIC_NRF
+	depends on !NRF_IRONSIDE_CALL
```

Comment:
Will this break if someone wants to use some software crypto on nrf54h20?

Reply:
software crypto on nrf54h20 is not supported. So if it does break, then that is intended behaviour :)